Library/Sender.ts

```diff
@@ -137,30 +137,32 @@ class Sender {
     }
 
     public saveOnCrash(payload: string) {
-        this._storeToDiskSync(payload);
+        if (this._enableOfflineMode) {
```

Comment:
Do you need similar check on reading from the storage or it was already there? As per PR title, it looks like we wanted to add both.